### PR TITLE
[v1.1] Renaming of AddressBook to fit context of project

### LINKED ISSUE #30: [v1.1] Renaming of AddressBook to fit context of project
```
AddressBook --> Project
Address --> Project
Person --> Task
```

Please merge all changes to `v1.1/rename-variables`

### PR DESCRIPTION
**Pull request by andreathniah**
All consolidated changes to morph AddressBook to NoteNote.

Closes #30

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ endif::[]
 _Marco Jakob_.
 * Libraries used: https://github.com/TestFX/TestFX[TextFX], https://github.com/FasterXML/jackson[Jackson], https://github.com/google/guava[Guava], https://github.com/junit-team/junit5[JUnit5]
 
-* This application was built using the source code found in https://github.com/se-edu/addressbook-level4[AddressBook-level4] by SE-EDU.
+* This application was built using the source code found in https://github.com/se-edu/addressbook-level4[Project-level4] by SE-EDU.
```
Comment (josephinetanadi) [inline README.adoc]: This should be reverted back to original.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 4 - User Guide
+= Project Level 4 - User Guide
```
Comment (josephinetanadi) [inline docs/UserGuide.adoc]: Shouldn't this be NoteNote?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -18,7 +18,7 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+Project Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
```
Comment (josephinetanadi) [inline docs/UserGuide.adoc]: This too. And replace AB4 with NN or remove?

File: docs/UsingTravis.adoc [hunk 4]
```diff
@@ -61,7 +61,7 @@ image:travis_build.png[Travis build]
    Using this account, generate a personal access token https://github.com/settings/tokens/new[here].
 +
 [NOTE]
-Personal access tokens are like passwords so make sure you keep them secret! If the personal access token is leaked, please delete it and generate a new one.
+Taskal access tokens are like passwords so make sure you keep them secret! If the personal access token is leaked, please delete it and generate a new one.
```
Comment (josephinetanadi) [inline docs/UsingTravis.adoc]: What is Taskal?

File: src/main/java/seedu/project/MainApp.java [hunk 5]
```diff
@@ -69,25 +69,27 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from {@code storage}'s address
+     * book and {@code userPrefs}. <br>
+     * The data from the sample address book will be used instead if
```
Comment (josephinetanadi) [inline src/main/java/seedu/project/MainApp.java]: I think we should do a mass replacement of "address book" to "project".

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -89,7 +89,7 @@ Set up Travis to perform Continuous Integration (CI) for your fork. See <<UsingT
 After setting up Travis, you can optionally set up coverage reporting for your team fork (see <<UsingCoveralls#, UsingCoveralls.adoc>>).
 
 [NOTE]
-Coverage reporting could be useful for a team repository that hosts the final version but it is not that useful for your personal fork.
+Coverage reporting could be useful for a team repository that hosts the final version but it is not that useful for your taskal fork.
```
Comment (gautamrajulu) [inline docs/DeveloperGuide.adoc]: Needs to be changed back to personal

Comment (gautamrajulu) [inline docs/DeveloperGuide.adoc]: revert to original

**Pull request merged**